lte/gateway/c/oai/tasks/sctp/sctp_primitives_server.c

```diff
@@ -133,21 +141,25 @@ static int handle_message(zloop_t* loop, zsock_t* reader, void* arg) {
 //------------------------------------------------------------------------------
 static void* sctp_thread(__attribute__((unused)) void* args_p) {
   itti_mark_task_ready(TASK_SCTP);
-  init_task_context(TASK_SCTP, (task_id_t[]){TASK_MME_APP, TASK_S1AP}, 2, handle_message, &sctp_task_zmq_ctx);
-  init_task_context(TASK_SCTP, (task_id_t[]){TASK_AMF_APP, TASK_NGAP}, 2, handle_message, &sctp_task_zmq_ctx);
+  init_task_context(
+      TASK_SCTP, (task_id_t[]){TASK_MME_APP, TASK_S1AP, TASK_NGAP}, 3,
+      handle_message, &sctp_task_zmq_ctx);
 
   zloop_start(sctp_task_zmq_ctx.event_loop);
   sctp_exit();
   return NULL;
 }
 
-int sctp_init(const mme_config_t* mme_config_p) {
-  OAILOG_DEBUG(LOG_SCTP, "Initializing SCTP task interface\n");
-
+int sctp_init(
+    const mme_config_t* mme_config_p /* , const amf_config_t* amf_config_p*/) {
   if (init_sctpd_downlink_client(!mme_config.use_stateless) < 0) {
     OAILOG_ERROR(LOG_SCTP, "failed to init sctpd downlink client\n");
   }
-
+  /*
```

Comment:
commented code without a reason. can you explain why you have commented code? what is your plan to test it?